chapters/intro-computer-architecture/c-goto-statements/drills/tasks/goto-algs/support/vector-max/vector-max.c

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include <stdio.h>
+
+int main(void)
+{
+	int v[] = {4, 1, 2, -17, 15, 22, 6, 2};
+	int max;
+	int i;
+
+	/* TODO: Implement finding the maximum value in the vector */
```

Comment:
I think replacing ```vector``` with ```array``` would better fit the description of the exercise : "1. Implement an algorithm for finding the largest element of an array".
```suggestion
	/* TODO: Implement finding the maximum value in the array */
```